src/pushsource/_impl/backend/koji_source.py

```diff
@@ -602,6 +606,7 @@ def _push_items_from_vmi_build(self, nvr, meta):
                     description="",
                     dest=self._dest,
                     src=item_src,
+                    boot_mode=boot_mode,
```

Comment:
In connection with above comment, note that `boot_mode` here is still a `str|None` and there doesn't seem to be any conversion to a `BootMode` happening.

I would expect this to crash in the validator on real data, and CI only passes because the test is buggy.

Reply:
You're right, I've fixed it now, PTAL again